src/templates/specific-project.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { Link, graphql } from 'gatsby'
+import Img from 'gatsby-image'
+import Layout from '../components/Layout.js'
+import Navbar from '../components/Navbar.js'
+import { Row, Col } from 'antd';
+import { Button } from 'antd';
+
+const ProjectTemplate = ({ data }) => (
+    <Layout>
+        <br></br>
+        <br></br>
+        <br></br>
+        <Row type="flex" justify="center" align="center" >
+            <Col xs={20} sm={10} m={10} l={12} xl={6}>
+              <Img fixed={data.strapiProject.project_image.childImageSharp.fixed} />
+            </Col>
+            <Col xs={20} sm={10} m={10} l={12} xl={6}>
+              <br></br>
+              <br></br>
+
+
+              <h1 className="has-text-weight-bold is-size-2">{data.strapiProject.project_name}</h1>
+              <p>
+              <hr style={{
+                    color:'#1C2833',
+                    backgroundColor: '#1C2833',
+                    height: 2
+                  }}/>
+                <Link style={{color: '#1C2833'}} to={'${data.strapiProject.id}'}>Sponsored by ...</Link>
+                <br></br>
+                <br></br>
+                <br></br>
+                <Button type="default" block>Contact Sponsor</Button>
+              </p>
+            </Col>
+        </Row>
+        <Row type="flex" justify="center" align="center">
```

Comment:
This and the following sections are just BEGGING to be component-ized. Also, I kinda liked the underlined headings from @liliedelamotte about page work...